tests/test_utils.py

```diff
@@ -56,6 +56,35 @@ def test_mount_and_chown(mocker, chown_config):
     )
 
 
+def test_activate_mount(mocker):
+    mock_mount = mocker.patch("ephemeral_storage_setup.utils.mount")
+    mock_add_to_fstab = mocker.patch("ephemeral_storage_setup.utils.add_to_fstab")
+    mock_populate_directory = mocker.patch("ephemeral_storage_setup.utils.populate_directory")
```

Comment:
As mentioned in other channels, I believe this type of tests have net negative value in the long term, as they force a straight-jacket on the real code, and can even become dangerous if more (destructive) operations are added in the real code, which are not caught by an appropriate mock.

Tried to investigate mocking the entire utils module, but couldn't find a way to make that work with a single exception for the method to be tested.